validation/cache.go

```diff
@@ -0,0 +1,35 @@
+package validation
+
+import (
+	"fmt"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+type SchemaCache struct {
+	cache map[string]*gojsonschema.Schema
+}
+
+func NewSchemaCache() *SchemaCache {
+	return &SchemaCache{make(map[string]*gojsonschema.Schema)}
+}
+
+// Get returns a JSON Schema of the passed URI.
+// It calls network once first time, and stores it in memory for later use.
+func (l *SchemaCache) Get(schemaURI string) (*gojsonschema.Schema, error) {
+	if loader, ok := l.cache[schemaURI]; ok {
+		return loader, nil
+	}
```

Comment:
```suggestion
	if schema, ok := l.cache[schemaURI]; ok {
		return schema, nil
	}
```
I think it would be better to call this `schema`.

Reply:
Thanks!
I fixed it.(https://github.com/medibloc/panacea-oracle/pull/105/commits/5140787477de4e7f10a518b3a05b16c105daf55d)